### ISSUE TITLE
Add message/warning to run sync when hosts on provider are changed after the last sync

### ISSUE THREAD

**Issue by valentinocossar** [beliven-it/hssh#28]
Add message/warning to run sync when hosts on provider are changed after the last sync. With this implementation, we aim to notify users when hosts are changed on the provider repository after they do the last sync, in this way a user knows when he has to run another sync to pull the latest hosts.

**Comment by CasvalDOT** [comment]
Okay, we can also add an extra option in the config file to allow user to set this behavior automatically.
What do you think @valentinocossar?
An example must be:
```
auto_sync_on_provider_change: true | false
```

Default can be false

**Comment by valentinocossar** [comment]
Yes, we can also add this feature. I would simplify the name with `auto_sync: true | false`. Default to false.
